<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestRunner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Images/Test2.png</normaloff>:/Images/Test2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelIntro">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Test Runner</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Please open a test.

CongChen@utdallas.edu
2012/3/12</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionNext"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="TestRunner.qrc">
     <normaloff>:/Images/OpenTest.png</normaloff>:/Images/OpenTest.png</iconset>
   </property>
   <property name="text">
    <string>Load Test</string>
   </property>
   <property name="toolTip">
    <string>Load test</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="TestRunner.qrc">
     <normaloff>:/Images/Start.png</normaloff>:/Images/Start.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="TestRunner.qrc">
     <normaloff>:/Images/Exit.png</normaloff>:/Images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="TestRunner.qrc"/>
 </resources>
 <connections/>
</ui>
